src/mame/drivers/minitel_2_rpic.cpp

```diff
@@ -92,6 +108,8 @@ class minitel_state : public driver_device
 		, m_ts9347(*this, "ts9347")
 		, m_palette(*this, "palette")
 		, m_i2cmem(*this, "i2cmem")
+		, m_modem(*this, "modport0")
+		, m_serport(*this, "serport0")
```

Comment:
Slot device tags are user-facing.  How are the ports actually labelled on the machine?

Reply:
"modport0" changed to "modem" and "serport0" to "periinfo" 
Please note that the real name for "periinfo" is  "prise péri-informatique".